### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by FatRat60**
Login and signup take you to Profile view where you can either edit profile, view pages, or view reviews. (Reviews not implemented yet) View pages takes you to a list of the pages associated with your account. You can then either click to add a new page or view one of the already created pages. When viewing a page you can opt to edit it.

### CODE REVIEW

**Review by PanyaOu — APPROVED** [review]
good add

File: src/TestLogin.js [hunk 1]
```diff
@@ -1,80 +1,89 @@
-import React, { useState } from 'react'
```
Comment (colewarner24) [inline src/TestLogin.js]: Delete testlogin and signup

**Review by colewarner24 — CHANGES_REQUESTED** [review]
merge updated main into code

**Review by colewarner24 — APPROVED** [review]
Code looks good go ahead and push

**Pull request closed without merging**